### PR TITLE
More fixes

### PR DESCRIPTION
**Pull request by till**
# Features

 - add variable to configure container networks (for linking, e.g. LDAP)
 - don't expose container by default on host port

# Bugfixes

 - made the endpoints check less prone to error (I think I broke it in #6)

# Maintenance

 - enabled more tests on CircleCI
 - refactored settings into its own task file (and left some debug code)

# BC/Change

 - removed obsolete `docker-py`
 - documented dependency instead and added example
 - TL;DR: use `docker` lib instead

# Tests

https://circleci.com/gh/pngmbh/ansible-role-portainer/19 ✅

### CODE REVIEW

File: molecule/default/playbook.yml [hunk 1]
```diff
@@ -15,4 +15,4 @@
     admin_user: admin
     admin_password: admin
     endpoints:
-      - {name: portainer-mgmt, url: "unix:///var/run/docker.sock"}
```
Comment (till) [inline molecule/default/playbook.yml:18]: FYI, `unix://` is no longer a "valid" protocol, according to the Portainer API.
